server/models/Collective.js

```diff
@@ -1451,7 +1451,9 @@ export default function (Sequelize, DataTypes) {
 
       case roles.MEMBER:
       case roles.ADMIN:
-        await this.sendNewMemberEmail(user, role, member, sequelizeParams);
+        if (this.type !== 'FUND') {
```

Comment:
Shouldn't that also apply to projects?

Reply:
We can do the same, I'll do it. But practically, it's not possible right now to invite people to the project.